src/wagtail_bynder/utils.py

```diff
@@ -24,11 +24,21 @@ def download_file(
     url: str, max_filesize: int, max_filesize_setting_name: str
 ) -> InMemoryUploadedFile:
     name = os.path.basename(url)
+    response = requests.get(url, timeout=20, stream=True)
+
+    # Make sure we don't store error responses instead of the file requested
+    if response.status_code != 200:
```

Comment:
non-blocking nitpick:

```suggestion
    if response.status_code != HTTPStatus.OK:
```

with `from http import HTTPStatus` for the import

Reply:
:+1: done in 8a06192e9801562e94c1331a62e9879585c4b6cc